scenarios/reproducers/delta-nfs-networking-definition.yml

```diff
@@ -0,0 +1,157 @@
+---
```

Comment:
Ok I think this one should simply be restored, and we should add the relevant networks to the uni* jobs that override anyway this.